Introducing IdpLinkAction as AIA to replace client-initiated account … (#38952)
keycloak/keycloak: docs/documentation/upgrading/topics/changes/changes-26_3_0.adoc

== Deprecation notices

This lists functionality that continues to work as before in this release, but will be removed in a future major release.

=== Deprecated proprietary protocol for client initiated linking to the identity provider account

> When you want the user, which is authenticated to your client application, to link his account to specified identity provider, it is now
> recommended to use the Application initiated action (AIA) based mechanism with the action `idp_link`. The proprietary custom protocol for client initiated account linking

Reviewer: ```suggestion
When you want the user, who is authenticated to your client application, to link his or her account to a specific identity provider, consider using the Application initiated action (AIA) based mechanism with the action `idp_link`. The proprietary custom protocol for client initiated account linking
```
